USER_GUIDE.md

```diff
@@ -137,6 +137,12 @@ extra_cmd_args = "--stepfactor 2"
 ```
 You can find more examples under `conf/test`. In a test schema file, you can adjust arguments as shown above. In the `cmd_args` section, you can provide different values other than the default values for each argument. In `extra_cmd_args`, you can provide additional arguments that will be appended after the NCCL test command. You can specify additional environment variables in the `extra_env_vars` section.
 
+#### Note : If you intend to run the llama model, it is important to execute the following steps :
+
+1. go to https://huggingface.co/docs/transformers/en/model_doc/llama#usage-tips
+2. follow the instructions under Usage Tip on how to download the tokenizer
+3. replace "training.model.tokenizer.model=TOKENIZER_MODEL" with "training.model.tokenizer.model=YOUR_TOKENIZER_PATH" (the tokenizer should be a .model file) in conf/v0.6/general/test/llama.toml
+
```

Comment:
First, your changes are breaking the original intention of the USER_GUIDE.md. Currently, USER_GUIDE.md describes how to add a new test template from a new Docker image. You are explaining how to run LLAMA in the middle of the guide. The best way I can think of is to break USER_GUIDE.md into two sections: adding a new test template and configuring NeMo Launcher.
Second, your PR title and description need to be updated accordingly. You are updating USER_GUIDE.md, not README.md.

Reply:
Yes, USER_GUIDE covers more than this, but what I meant was that the current flow does not focus on any individual test template. Therefore, we need to expand the USER_GUIDE to include more information. While USER_GUIDE provides general guidance, adding LLAMA instructions in the middle of the document does not address the issue with the current flow.